src/components/home/StoryUpload.jsx

```diff
@@ -262,6 +262,7 @@ const StoryUpload = ({ onSuccess = () => {} }) => {
                         name="storyWorldLead"
                         id="storyWorldLead"
                         className="bg-gray-50 w-full md:w-[35vw] border p-2 border-gray-300 text-gray-900 sm:text-md rounded-lg focus:ring-primary-600 focus:border-primary-600 block"
+                        disabled
```

Comment:
this should be disabled only if backend given name property or whatever that we are setting for the storyworld lead exists